common/src/main/java/com/wynntils/models/stats/StatListOrderer.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.stats;
+
+import com.wynntils.models.stats.builders.MiscStatKind;
+import com.wynntils.models.stats.type.DamageStatType;
+import com.wynntils.models.stats.type.DefenceStatType;
+import com.wynntils.models.stats.type.MiscStatType;
+import com.wynntils.models.stats.type.SpellStatType;
+import com.wynntils.models.stats.type.StatListDelimiter;
+import com.wynntils.models.stats.type.StatListOrdering;
+import com.wynntils.models.stats.type.StatType;
+import com.wynntils.utils.ListUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public final class StatListOrderer {
+    // Legacy order was defined by Athena as a fixed list. This was missing several stat types; I have
+    // tried filling them in into "logical" places
+    private static final List<String> LEGACY_ORDER = List.of(
```

Comment:
Shouldn't this look more sane if we had `List<List<String>>` where there is a delimiter after each inner list? Although this is just purely looks.

Reply:
My idea was that we should allow users to provide a `CUSTOM` ordering, in addition to the three predefined orderings by us. My plan was that this should take the shape of a json file containing an array of strings, just like here, where the "" were used to mark delimiters. 

But now when I think of it, of course we could even there make it a list of lists.

Ugh. I dunno. Maybe it would be better, but ... I'm not sure. I think this also gives a nice visual indication of where the separator goes. Like you know when you set up a toolbar and you drag in a "delimiter space" between buttons. 

If you are okay with this I'd rather not change it. If nothing else because I'm starting to get sick and tired on tweaking this code.

Reply:
Okay, it can stay.